include.js

```diff
@@ -3018,6 +3018,14 @@ let cards = [
     author: 'Samuel Luiz',
     githubLink: 'https://github.com/samluiz'
   }
```

Comment:
Hello @kareninarg , you seemed to have omitted something here, can you guess?

Reply:
Thanks, I think I fixed it. Please take a look